### PR TITLE
Obfuscating Dart code

### PR DESCRIPTION
**Pull request by Sfshaza**
A continuation of PR https://github.com/flutter/website/pull/1071.

* As Chris requested, the obfuscation details are now in the [Flutter wiki.](https://github.com/flutter/flutter/wiki/Obfuscating-Dart-Code).
* Incorporated feedback from @mit-mit, @Ehud-Banunu, and @xster.

### CODE REVIEW

File: animations/index.md [hunk 1]
```diff
@@ -130,7 +130,11 @@ classes.
 Animations that are broken into smaller motions, where some of the motion is delayed.
 The smaller animations may be sequential, or may partially or completely overlap.
 
-* [Staggered Animations](/animations/staggered-animations/) <img src="/images/ic_new_releases_black_24px.svg" alt="this doc is new!"> NEW<br>
+* [Staggered Animations](/animations/staggered-animations/)
+
+<!-- Save so I can remember how to add it back later.
+<img src="/images/ic_new_releases_black_24px.svg" alt="this doc is new!"> NEW<br>
+-->
```
Comment (cbracken) [inline animations/index.md]: Was this comment intended to be checked in?

**Comment by cbracken** [comment]
lgtm

File: android-release-build.md [hunk 2]
```diff
@@ -161,8 +164,9 @@ Create `/android/app/proguard-rules.pro` file and add rules listed below.
 -keep class io.flutter.plugins.**  { *; }
 ```
 
-The configuration above will only protect Flutter libraries.
-Any additional libraries e.g. Firebase will require their own rules to be added.
+The configuration above only protects Flutter libraries.
```
Comment (xster) [inline android-release-build.md]: I'd specify 'Flutter engine libraries' to avoid confusion

File: animations/index.md (hunk 1 above)
Comment (Sfshaza) [reply animations/index.md]: Actually, yes.

File: android-release-build.md (hunk 2 above)
Comment (Sfshaza) [reply android-release-build.md]: Done. Thanks, @xster

**Pull request merged**